### PR TITLE
Support the output of timestamps per result

### PR DESCRIPTION
**Pull request by joachimvh**

### CODE REVIEW

File: bin/runner.js [hunk 1]
```diff
@@ -6,7 +6,11 @@ const url = require('url');
 
 const args = minimist(process.argv.slice(2));
 if (args._.length !== 2 || args.h || args.help) {
-  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [--replication 5] [--warmup 0] [--output output.csv]\n');
+  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [-r 5] [-w 0] [-o output.csv] [--timestamps]\n');
+  process.stderr.write('  -r # of replication runs\n');
+  process.stderr.write('  -w # of warmup runs\n');
+  process.stderr.write('  -o output file\n');
+  process.stderr.write('  --timestamps if enabled also outputs timestamps for when each result was received\n');
```
Comment (rubensworks) [inline bin/runner.js]: Perhaps you can use `-t` here for consistency?

File: bin/runner.js [hunk 2]
```diff
@@ -6,7 +6,11 @@ const url = require('url');
 
 const args = minimist(process.argv.slice(2));
 if (args._.length !== 2 || args.h || args.help) {
-  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [--replication 5] [--warmup 0] [--output output.csv]\n');
+  process.stderr.write('usage: sparql-benchmark-runner endpoint-url path-to-queries [-r 5] [-w 0] [-o output.csv] [--timestamps]\n');
+  process.stderr.write('  -r # of replication runs\n');
```
Comment (rubensworks) [inline bin/runner.js]: I would add a couple of spaces between param name and description, it reads a bit confusing now.

File: bin/runner.js [hunk 3]
```diff
@@ -113,6 +133,7 @@ function stop(hrstart) {
 async function isUp() {
   return new Promise((resolve) => {
     const req = http.request(options, res => {
+      console.log(res.statusCode);
```
Comment (rubensworks) [inline bin/runner.js]: Unneeded?

Comment (joachimvh) [reply bin/runner.js]: woops

**Comment by joachimvh** [comment]
Yea adding it to a separate file is also an option, wanted to stay in line with what the tool was already doing. Not sure how much difference it would make, it is now doing what the tool down the line would be doing anyway. But it might be cleaner I guess

File: bin/runner.js (hunk 1 above)
Comment (joachimvh) [reply bin/runner.js]: yea was doubting, wanted to make it special since it's a boolean value instead of one with an additional parameter

Comment (rubensworks) [reply bin/runner.js]: I have no strong opinion, either way is fine for me :-)

**Comment by joachimvh** [comment]
updated

**Pull request merged**